WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroViewModel.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+/// View model for `BlazeCreateCampaignIntroView`
+final class BlazeCreateCampaignIntroViewModel: ObservableObject {
+    private let analytics: Analytics
+
+    @Published var showLearHowSheet: Bool = false
```

Comment:
nit: typo: "showLearnHowSheet"

Reply:
Nice catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11583/commits/4d0269eb6eb0a227979258b57a712dba3e85f739